swagger.yaml

```diff
@@ -274,9 +274,12 @@ definitions:
 
   ApplicationPatch:
     type: 'object'
+    description: 'One of status OR validations'
     properties:
       status:
         type: 'string'
+      validations:
+        type: 'string'
```

Comment:
It's going to be an `object`, is it easier for you to have a `string`?

Not a big deal, I will just need to stringify and parse it.

Reply:
If we have string, I can simply save and return. Happy to switch it though.
 WDYT?

Reply:
Ok then, let's leave the whole logic in the FE 👍